server/app/assets/javascripts/main.ts

```diff
@@ -344,6 +344,20 @@ function configurePredicateFormOnOperatorChange(event: Event) {
   )
 }
 
+function attachFormDebouncers() {
+  Array.from(document.querySelectorAll('.cf-debounced-form')).forEach((el) => {
+    const submitEl = el.querySelector('button[type="submit"]')
```

Comment:
Suggestion: Rename el to formEl for added clarity.

Reply:
Done!